inference-engine/tests/functional/inference_engine/ir_serialization/serialize.cpp

```diff
@@ -48,18 +58,18 @@ TEST_P(SerializationTest, CompareFunctions) {
 }
 
 INSTANTIATE_TEST_CASE_P(IRSerialization, SerializationTest,
-        testing::Values(std::make_tuple("add_abc.xml"),
-                        std::make_tuple("add_abc_f64.xml"),
-                        std::make_tuple("split_equal_parts_2d.xml"),
-                        std::make_tuple("addmul_abc.xml"),
-                        std::make_tuple("add_abc_initializers.xml"),
-                        std::make_tuple("experimental_detectron_roi_feature_extractor.xml"),
-                        std::make_tuple("experimental_detectron_detection_output.xml"),
-                        std::make_tuple("nms5.xml"),
-                        std::make_tuple("shape_of.xml")));
+        testing::Values(std::make_tuple("add_abc.xml", "add_abc.bin"),
+                        std::make_tuple("add_abc_f64.xml", ""),
+                        std::make_tuple("split_equal_parts_2d.xml", "split_equal_parts_2d.bin"),
+                        std::make_tuple("addmul_abc.xml", "addmul_abc.bin"),
+                        std::make_tuple("add_abc_initializers.xml", "add_abc_initializers.bin"),
+                        std::make_tuple("experimental_detectron_roi_feature_extractor.xml", ""),
+                        std::make_tuple("experimental_detectron_detection_output.xml", ""),
+                        std::make_tuple("nms5.xml", "nms5.bin"),
```

Comment:
Why do we need to specialize the bin files?

Reply:
It is a matter of usage, this way better explains idea of serialization test without exploring how it works inside, behavior is the same.